scripts/create_channel.go

```diff
@@ -0,0 +1,67 @@
+package scripts
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/Brightscout/mattermost-load-test-scripts/serializers"
+	"github.com/Brightscout/mattermost-load-test-scripts/utils"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+func CreateChannel(config *serializers.Config) error {
+	client := model.NewAPIv4Client(config.ConnectionConfiguration.ServerURL)
+	if _, lErr := client.Login(config.ConnectionConfiguration.AdminEmail, config.ConnectionConfiguration.AdminPassword); lErr.Error != nil {
+		return errors.New(lErr.Error.Message)
+	}
+
+	var newChannels []*serializers.ChannelResponse
+	response, err := utils.LoadResponse()
+	if err != nil {
+		return err
+	}
+
+	for _, channel := range config.ChannelsConfiguration {
+		team, tErr := client.GetTeamByName(channel.MMTeamName, "")
+		if tErr.Error != nil {
+			return errors.New(tErr.Error.Message)
+		}
+
+		createdChannel, cErr := client.CreateChannel(&model.Channel{
+			TeamId:      team.Id,
+			Name:        channel.Name,
+			DisplayName: channel.DisplayName,
+			Type:        channel.Type,
+		})
+
+		if cErr.Error != nil {
+			return errors.New(cErr.Error.Message)
+		}
+
+		newChannels = append(newChannels, &serializers.ChannelResponse{
+			ID: createdChannel.Id,
+		})
+
+		newUserIDs := []string{}
+		for _, user := range response.UserResponse {
+			newUserIDs = append(newUserIDs, user.ID)
+		}
+
+		if _, aErr := client.AddTeamMembers(team.Id, newUserIDs); aErr.Error != nil {
+			return errors.New(aErr.Error.Message)
+		}
+
+		channelLinkCommand := fmt.Sprintf("/msteams-sync link %s %s", channel.MSTeamsTeamID, channel.MSTeamsChannelID)
+		if _, eErr := client.ExecuteCommand(createdChannel.Id, channelLinkCommand); eErr.Error != nil {
```

Comment:
I think we should avoid creating so many variable for a error of same type and create a single err with some meaningful name and use it everywhere

Reply:
Fix everywhere if changing